### DISCUSSION TITLE
titleTemplate over the whole page

### DISCUSSION THREAD

**Discussion by rickhaer** [unjs/unhead#406] [Q&A]
Hello, I'm wondering if I'm doing something wrong. I would like to make sure that I always set a title in my page components and that this title automatically gets ‘| MySiteName’ added at the end.

For example on my contact component I write 

```
<script setup>
import { useHead } from ‘@unhead/vue’;
useHead({
    title: ‘Contact’,
});
</script>
```

But my title is set as ‘Contact | MySiteName’. 

I already found something about titleTemplates here https://unhead.unjs.io/usage/guides/title-template but it does not seems to work.

I tried to set it in my App.vue like:

```
<script setup>
import { useHead } from ‘@unhead/vue’;
useHead({
    titleTemplate: ‘%s | MySiteName’,
});
</script>
```

but it still just uses 'Contact' without my template.

I also tried to set it in my main.js file directly into my `const head = createHead();` but it did also not work

Am i missing something?
Thanks for helping!

**Comment by harlan-zw** [comment]
Hi, this is the correct usage. Could you try and make a reproduction of the issue? https://stackblitz.com/edit/vitejs-vite-uijgqa?file=package.json

**Comment by rickhaer** [comment]
Hi @harlan-zw, thanks for your answer. I started recreating this issue in stackblitz based on my code. It actually worked in stackblitz and after checking on my local machine again, it also worked there too ... so this can be marked as done! Thanks anyways :)
